test/api_compatibility.js

```diff
@@ -56,47 +40,43 @@ module.exports = function apiCompatibilityChecks(createTracer) {
-                    var tracer = new opentracing.Tracer();
-                    var span = tracer.startSpan('test_operation');
-                    expect(function() { tracer.startSpan('child', { childOf : span }); }).to.not.throw(Error);
-                });
+        describe('Tracer#startSpan', function() {
+            it('should handle Spans and SpanContexts', function() {
+                var tracer = new opentracing.Tracer();
+                var span = tracer.startSpan('test_operation');
+                expect(function() { tracer.startSpan('child', { childOf : span }); }).to.not.throw(Error);
             });
+        });
 
-            describe('Tracer#inject', function() {
-                it('should enforce the required carrier types', function() {
-                    var tracer = new opentracing.Tracer();
-                    var spanContext = tracer.startSpan('test_operation').context();
-                    var textCarrier = {};
-                    expect(function() { tracer.inject(spanContext, opentracing.FORMAT_TEXT_MAP, textCarrier); }).to.not.throw(Error);
-                    expect(function() { tracer.inject(spanContext, opentracing.FORMAT_TEXT_MAP, ''); }).to.throw(Error);
-                    expect(function() { tracer.inject(spanContext, opentracing.FORMAT_TEXT_MAP, 5); }).to.throw(Error);
-
-                    var binCarrier = new opentracing.BinaryCarrier();
-                    expect(function() { tracer.inject(spanContext, opentracing.FORMAT_BINARY, binCarrier); }).to.not.throw(Error);
-                    expect(function() { tracer.inject(spanContext, opentracing.FORMAT_BINARY, new Object); }).to.not.throw(Error);
-                    expect(function() { tracer.inject(spanContext, opentracing.FORMAT_BINARY, {}); }).to.not.throw(Error);
-                    expect(function() { tracer.inject(spanContext, opentracing.FORMAT_BINARY, { buffer : null }); }).to.not.throw(Error);
-
-                    expect(function() { tracer.extract(opentracing.FORMAT_BINARY, binCarrier); }).to.not.throw(Error);
-                    expect(function() { tracer.extract(opentracing.FORMAT_BINARY, {}); }).to.not.throw(Error);
-                    expect(function() { tracer.extract(opentracing.FORMAT_BINARY, { buffer : null }); }).to.not.throw(Error);
-                    expect(function() { tracer.extract(opentracing.FORMAT_BINARY, { buffer : '' }); }).to.throw(Error);
-                    expect(function() { tracer.extract(opentracing.FORMAT_BINARY, { buffer : 5 }); }).to.throw(Error);
-                });
-                it('should handle Spans and SpanContexts', function() {
-                    var tracer = new opentracing.Tracer();
-                    var span = tracer.startSpan('test_operation');
-                    var textCarrier = {};
-                    expect(function() { tracer.inject(span, opentracing.FORMAT_TEXT_MAP, textCarrier); }).to.not.throw(Error);
-                });
+        describe('Tracer#inject', function() {
```

Comment:
can we not keep the same nesting/identation to miminize the diff? What do we gain by the change in nesting?